src/main/java/org/jenkinsci/plugins/ghprb/extensions/comments/GhprbCommentFile.java

```diff
@@ -60,9 +60,11 @@ public String postBuildComment(Run<?, ?> build, TaskListener listener) {
                     content = FileUtils.readFileToString(new File(scriptFilePathResolved));
                 }
 
-                msg.append("Build comment file: \n--------------\n");
-                msg.append(content);
-                msg.append("\n--------------\n");
+                if (content.length() > 0) {
```

Comment:
Should we trim spaces and newlines since they’re not human readable?

Reply:
No worries.  I'm fine without the trimming.  I was just curious what your thoughts were.  Thanks.